next/@types/network.ts

```diff
@@ -29,24 +30,33 @@ export interface Atlas {
   version: string;
 }
 
+export type DatasetQueryOrgan = string;
+
 export interface Network {
   atlases: Atlas[];
   contact: Contact;
   coordinators: Coordinator[];
+  datasetQueryOrgans: DatasetQueryOrgan[];
   key: NetworkKey;
   name: string;
   path: string;
 }
 
 export type NetworkParam = {
   network: Network;
+  projects: ProjectsEntityResponse[];
```

Comment:
Hi @BruceRodrigues, we may need to use an aggregated response for example `disease` from `AggregatedDonorOrganismsResponse` aka `donorOrganisms`.

I think change all uses of `ProjectsEntityResponse` to `ProjectsResponse` here and elsewhere, and update the property on NetworkParam and also the NetworkContext to something like `projectsResponses` so there is no confusion over the `projects` property in `ProjectsEntityResponse`.